### PR TITLE
Add support for custom nsp formatters

### LINKED ISSUE #19: Cannot use thirdparty formatter
```
npm i grunt-nsp nsp-formatter-checkstyle
```

gruntfile:

```
...
           nsp: {
                package: grunt.file.readJSON('package.json'),
                shrinkwrap: grunt.file.readJSON('npm-shrinkwrap.json'),
                output: 'checkstyle',
            },
...
```

I would expect this to work, but instead i see:

`Invalid formatter specified in config. Must be one of codeclimate,default,summary,json,none`

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #19

### CODE REVIEW

File: tasks/nsp.js [hunk 1]
```diff
@@ -41,13 +35,7 @@ module.exports = function (grunt) {
...
 
     if (grunt.option('output')) {
-
-      if (Nsp.formatters.hasOwnProperty(grunt.option('output'))) {
-        formatter = Nsp.formatters[grunt.option('output')];
-      }
-      else {
-        grunt.log.write('Invalid formatter specified in options. Must be one of ' + Object.keys(Nsp.formatters).join(',') + '\n');
-      }
+      formatter = Nsp.getFormatter(grunt.option('outout'));
```
Comment (nlf) [inline tasks/nsp.js]: `outout` should be `output` probably :)

Comment (pdehaan) [inline tasks/nsp.js]: Doh, good catch... Amended and force pushed.

**Comment by nlf** [comment]
:+1:

**Comment by nlf** [comment]
just published grunt-nsp@2.3.0 with this fix (and the shrinkwrap removed)

**Comment by pdehaan** [comment]
Thanks @nlf, I'll see if I can fix the **gulp-nsp** issue today too so it works with custom formatters.

**Comment by nlf** [comment]
That'd be great, thanks @pdehaan

**Pull request merged**